proto/minder/v1/minder.proto

```diff
@@ -1244,13 +1265,23 @@ message UpstreamRepositoryRef {
     bool registered = 5;
 }
 
+// Repository API objects. This is only used in responses.
 message Repository {
     reserved 15;
     reserved "registered";
-    optional string id = 1; // This is optional when returning remote repositories
+    // id is the unique identifier of the repository within Minder.
+    // It is always populated, but the optional keyword is used for
+    // backwards compatibility.
+    optional string id = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
```

Comment:
We only use this on output, right?

Reply:
Yes, the `REQUIRED` tag works for outputs as well, letting the client know that the field will always be populated.